src/VehicleSelect/VehicleSelect.stories.tsx

```diff
@@ -0,0 +1,94 @@
+import { Meta, StoryFn } from "@storybook/react";
+import { SelectedVehicle, VehicleSelectProps } from "./VehicleSelect.types";
+
+import React from "react";
+import VehicleSelect from "./VehicleSelect";
+import { action } from "@storybook/addon-actions";
+import { useArgs } from "@storybook/client-api";
+
+/**
+ * Story metadata
+ */
+const meta: Meta<typeof VehicleSelect> = {
+  component: VehicleSelect,
+  title: "General/VehicleSelect"
+};
+export default meta;
+
+const Template: StoryFn<VehicleSelectProps> = args => {
+  // useArgs is a hook that returns the current state of the args object
+  const [{ selectedVehicles }, updateArgs] = useArgs<VehicleSelectProps>();
+
+  // update the args object with the new selectedVehicles value
+  React.useEffect(() => {
+    updateArgs({ selectedVehicles });
+  }, [selectedVehicles, updateArgs]);
+
+  // callback for when the selected vehicles change
+  const onVehicleChange = (selectedVehicle: SelectedVehicle[]) => {
+    updateArgs({ selectedVehicles: selectedVehicle });
+    action("onVehicleChange")(selectedVehicle);
+  };
+  return (
+    <VehicleSelect
+      {...args}
+      onVehicleChange={onVehicleChange}
+      selectedVehicles={selectedVehicles}
+    />
+  );
+};
+
+// Default story
+export const Default = {
```

Comment:
Add defaults for all the props missed adding for flexDirection and flexWrap.

Reply:
Added